ParallelPrimitives/RadixSort.cpp

```diff
@@ -68,7 +68,7 @@ void RadixSort::exclusiveScanCpu( int* countsGpu, int* offsetsGpu, const int nWG
 	OrochiUtils::waitForCompletion();
 }
 
-void RadixSort::compileKernels( oroDevice device, const std::string& kernelPath, const std::string& includeDir ) noexcept
+void RadixSort::compileKernels( oroDevice device, OrochiUtils* oroutils, const std::string& kernelPath, const std::string& includeDir ) noexcept
```

Comment:
I'd suggest using reference since this cannot be null and yet we do not check for `nullptr` in the code.
```suggestion
void RadixSort::compileKernels( oroDevice device, OrochiUtils& oroutils, const std::string& kernelPath, const std::string& includeDir ) noexcept
```